<launch>
	<include file="$(find assign1_2013)/launch/beacons/trial.launch" />
	<include file="$(find assign1_2013)/launch/path.launch" />
	
    <node pkg="assign1_2013" type="detector" name="image_converter" output="screen">
    	<remap from="in" to="/camera/rgb/image_color" />
    </node>
    <node pkg="assign1_2013" type="visual" name="field" output="screen" >
    </node>
  	<node pkg="assign1_2013" type="simple_nav" name="simple_nav" output="screen">
    </node>
   	<node pkg="assign1_2013" type="safety_first" name="safety_first" output="screen">
    </node>



	<node pkg="robot_pose_ekf" type="robot_pose_ekf" name="robot_pose_ekf">
	    <param name="output_frame" value="odom_combined"/>
	    <param name="freq" value="30.0"/>
	    <param name="sensor_timeout" value="1.0"/>
	    <param name="odom_used" value="true"/>
	    <param name="imu_used" value="true"/>
	    <param name="vo_used" value="true"/>
	    <param name="debug" value="false"/>
	    <param name="self_diagnose" value="false"/>
	    <param name="pub_odometry" value="true"/>
	    <!--remap from="vo" to="camera_rgb_optical_frame"/-->
	</node>

    <node pkg="image_view" type="image_view" name="field_view" />
    <node pkg="image_view" type="image_view" name="camera_view" >
      <remap from="image" to="/camera/rgb/image_color" />
    </node>

</launch>
